keyboards/nack/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,65 @@
+       KC_ESC,      KC_A,        KC_S,        KC_D,        KC_F,        KC_G,        KC_H,        KC_J,        KC_K,        KC_L,        KC_SCLN,     KC_QUOT,     KC_NUHS,
+//  | |---SHIFT----|---Z--------|---X--------|----C-------|-----V------|-----B------|-----N------|------M-----|---( ,; )---|---( .: )---|---( -_ )---|----UP------|-BACKSPACE--|
+       KC_LSFT,     KC_Z,        KC_X,        KC_C,        KC_V,        KC_B,        KC_N,        KC_M,        KC_COMM,     KC_DOT,      KC_SLSH,     KC_UP,       KC_BSPC,
+//  | |---CTRL-----|---ALT------|---META-----|----FN_1----|----------SPACE----------|----FN_2----|----AltGr---|--( <>| )---|---ENETER---|-LEFT-------|---DOWN-----|--RIGHT-----|
+       KC_LCTL,     KC_LALT,     KC_LGUI,     MO(NUM),     KC_SPC,      KC_NO,       MO(FN),      KC_RALT,     KC_NUBS,     KC_ENT,     KC_LEFT,      KC_DOWN,     KC_RIGHT
+//   \|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|
+    ),
+    [NUM] = LAYOUT(
+/*   __________________________________________________________________________________________________________________________________________________________________________
+    | \            \            \            \            \            \            \            \            \            \            \            \            \            \ */
+//  | |---TAB------|---( 1! )---|--( 2"@ )---|--( 3#£ )---|---( 4¤$ )--|--( 5%€ )---|---( 6& )---|--( 7/{ )---|--( 8([ )---|--( 9)] )---|--( 0=} )---|---( +?\ )--|---( ´` )---|
+       _______,     KC_1,        KC_2,        KC_3,        KC_4,        KC_5,        KC_6,        KC_7,        KC_8,        KC_9,        KC_0,        KC_MINS,     KC_EQL,
+//  | |----ESC-----|-----F1-----|-----F2-----|----F3------|----F4------|----F5------|-----F6-----|-----F7------|----F8-----|-----F9-----|-----F10----|----F11-----|----F12-----|
+       _______,     KC_F1,       KC_F2,       KC_F3,       KC_F4,       KC_F5,       KC_F6,       KC_F7,       KC_F8,       KC_F9,       KC_F10,      KC_F11,      KC_F12,
+//  | |---SHIFT----|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|
+       _______,     KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,
+//  | |---CTRL-----|---ALT------|---META-----|---FN_1-----|------------|------------|----FN_2----|--AltGr-----|---( §½ )---|------------|------------|------------|------------|
+       _______,     _______,     _______,     MO(FN),      KC_NO,       KC_NO,       TO(BASE),    _______,     KC_GRV,      KC_NO,       KC_NO,       KC_NO,       KC_NO
```

Comment:
https://github.com/vial-kb/vial-qmk/blob/52e5b55a4aec0653df9cd0f5709b0d9b4ea7ce66/docs/feature_layers.md?plain=1#L9

```suggestion
       _______,     _______,     _______,     KC_TRNS,      KC_NO,       KC_NO,       TO(BASE),    _______,     KC_GRV,      KC_NO,       KC_NO,       KC_NO,       KC_NO
```
I don't really get the use of `TO(BASE)` here since you only use momentary layer holds, but that's at least not wrong.